src/cpp/server/secure_server_credentials.cc

```diff
@@ -23,6 +23,7 @@
 #include <grpcpp/impl/codegen/slice.h>
 #include <grpcpp/security/auth_metadata_processor.h>
 
+#include "include/grpc/impl/codegen/log.h"  // for GPR_ASSERT
```

Comment:
Please remove the comment on this line as we don't usually do that.
Also, we usually do `<grpc/impl/codegen/log.h>`